### PR TITLE
added space between the title and the content of the notification and…

### PR DESCRIPTION
**Pull request by ImJimMoriarty**
… made the table responsive

## Short description
The card's title of the NotificationPaidDetail was too close to the notification's alert and the table inside was not responsive.

## List of changes proposed in this pull request

- Added space between the alert and the card's title (NotificationPaidDetail)
- Made responsive the table of CustomTableRow, now it changes the display property.

## How to test
You can check the result inside the detail of the notification where the status is "Paid".

### CODE REVIEW

File: packages/pn-personafisica-webapp/src/component/Notifications/NotificationPayment.tsx [hunk 1]
```diff
@@ -457,7 +457,7 @@ const NotificationPayment: React.FC<Props> = ({
     >
       <Paper sx={{ p: 3, mb: '1rem' }} className="paperContainer">
         <Grid container direction="row" justifyContent="space-between">
-          <Grid item xs={8} lg={8}>
+          <Grid item xs={8} lg={8} sx={{ marginBottom: '1rem' }}>
```
Comment (AndreaCimini90) [inline packages/pn-personafisica-webapp/src/component/Notifications/NotificationPayment.tsx:460]: Don't use rem or px or others, but use integer like 1 (= 8px), 2 (= 16px) and so on.
For this case, from figma, we have 16px and so we must use 2

`<Grid item xs={8} lg={8} mb={2}>`

File: packages/pn-commons/src/components/CustomTableRow.tsx [hunk 2]
```diff
@@ -7,11 +7,17 @@ type Props = {
 };
 
 const CustomTableRow = ({ label, value, dataTestId }: Props) => (
-  <TableRow sx={{ '& td': { border: 'none' }, verticalAlign: 'top' }} data-testid={dataTestId}>
-    <TableCell padding="none" sx={{ py: 1 }} data-testid="label">
+  <TableRow sx={{
+    paddingBottom: { xs: '1rem' },
```
Comment (AndreaCimini90) [inline packages/pn-commons/src/components/CustomTableRow.tsx:11]: As in the comment below, use 2 instead of 1rem

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: packages/pn-personafisica-webapp/src/component/Notifications/NotificationPayment.tsx (hunk 1 above)
Comment (ImJimMoriarty) [reply packages/pn-personafisica-webapp/src/component/Notifications/NotificationPayment.tsx]: Changed.

File: packages/pn-commons/src/components/CustomTableRow.tsx (hunk 2 above)
Comment (ImJimMoriarty) [reply packages/pn-commons/src/components/CustomTableRow.tsx]: Changed.

**Review by AndreaCimini90 — APPROVED** [review]

**Review by AndreaCimini90 — APPROVED** [review]

**Pull request merged**